### ISSUE TITLE
SimpleWorker does not use new version of code

### ISSUE THREAD

**Issue by louisabraham** [rq/rq#1434]
After changing the code of a function, SimplerWorker still uses the previous code.
I understand it's not a bug but it should be documented.

**Comment by selwin** [comment]
I have actually not tested this myself. Are you implying that RQ's default worker would use the new version of the code whenever there's code changes, but not `SimpleWorker`?

**Comment by louisabraham** [comment]
Yes. It's because the default worker spawns a new process and imports all modules again whereas SimpleWorker is typically used to avoid reimporting every time.
